client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -120,31 +160,120 @@ class TransferDomainPrecheck extends React.PureComponent {
 							<strong>{ heading }</strong>
 							{ isStepFinished && stepStatus }
 						</div>
-						{ isAtCurrentStep && (
-							<div>
-								<div className="transfer-domain-step__section-message">{ message }</div>
-								<div className="transfer-domain-step__section-action">
-									<Button
-										compact
-										onClick={ unlocked ? this.showNextStep : this.refreshStatus }
-										busy={ loading }
-									>
-										{ buttonText }
-									</Button>
-									{ stepStatus }
+						{ isAtCurrentStep &&
+							message && (
+								<div>
+									<div className="transfer-domain-step__section-message">{ message }</div>
+									{ buttonText && (
+										<div className="transfer-domain-step__section-action">
+											<Button
+												compact
+												onClick={ unlocked ? this.showNextStep : this.refreshStatus }
+												busy={ loading }
+											>
+												{ buttonText }
+											</Button>
+											{ stepStatus }
+										</div>
+									) }
 								</div>
-							</div>
-						) }
+							) }
 					</div>
 				</div>
 			</Card>
 		);
 	}
 
+	getRegistrationPeriodMessage() {
+		const { translate } = this.props;
+		const {
+			creationDate,
+			currentStep,
+			daysTransferLocked,
+			expiryDate,
+			expiryDateNew,
+			expiryDateOk,
+			inInitialRegistrationPeriod,
+			loading,
+			termMaximumInYears,
+			transferEligibleDate,
+		} = this.state;
+		const step = 1;
+		const isStepFinished = currentStep > step;
+		const dateFormat = translate( 'MMM D, YYYY', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'D MMMM YYYY', { context: 'momentjs format string (for LL)'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/check-initial-registration-period-during-transfer-in-precheck)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).